lib/model/rwfolder.go

```diff
@@ -1454,6 +1456,7 @@ func (f *sendReceiveFolder) performFinish(state *sharedPullerState) error {
 			// we have resolved the conflict.
 
 			state.file.Version = state.file.Version.Merge(state.version)
+			f.lastModBy = state.file.ModifiedBy.String()
 			if err = osutil.InWritableDir(f.moveForConflict, state.realName); err != nil {
```

Comment:
still not working, Now I'm getting:
have too many args to return   have (error)   want ()

			`state.file.Version = state.file.Version.Merge(state.version)
			err = osutil.InWritableDir(func(name string) {
				return f.moveForConflict(name, state.file.ModifiedBy.String())
			}, state.realName)
			if err != nil {
				return err
			}`